src/sentry/lang/native/sources.py

```diff
@@ -328,9 +376,6 @@ def validate_sources(sources, schema=SOURCES_SCHEMA):
     try:
         jsonschema.validate(sources, schema)
     except jsonschema.ValidationError as e:
-        if sources.get("type") == "appStoreConnect":
-            raise InvalidSourcesError("appStoreConnect is being decomissioned")
-
```

Comment:
I believe this check could never trigger because `sources` is a list of sources rather than an individual source and as such doesn't have a `type` key.